src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/JpaTimelinePersistenceAdapter.kt

```diff
@@ -55,6 +60,12 @@ class JpaTimelinePersistenceAdapter(
   override fun getTimelineForPrisoner(prisonNumber: String): Timeline? {
     log.info { "Getting Timeline for prisoner [$prisonNumber]" }
 
+    // TODO RR-566 - remove this temporary code
```

Comment:
I like this for testing purposes 👍 
Whether this is where we'd want to do it once we get further with the development, I'm not sure 🤔 
IE. Is it the job of a persistence adapter to be injected with a webclient and make API calls?

But totally works and OK for what you are trying to do here and now 👍

Reply:
100% not what we want - it's just temporary. I mentioned an upcoming service class in one of my other comments: https://github.com/ministryofjustice/hmpps-education-and-work-plan-api/pull/231/files#r1446120819